pkg/sql/sem/tree/schema_helpers.go

```diff
@@ -33,3 +33,26 @@ func IsSetOrResetSchemaLocked(n Statement) bool {
 	}
 	return false
 }
+
+// IsAllowedLDRSchemaChange returns true if the schema change statement is
+// allowed to occur while the table is being referenced by a logical data
+// replication job as a destination table.
+func IsAllowedLDRSchemaChange(n Statement) bool {
+	switch s := n.(type) {
+	case *CreateIndex:
+		return true
+	case *DropIndex:
+		return true
+	case *AlterTable:
+		// Allow adding a nullable column without a default value if that is the
```

Comment:
could we just merge CreateIndex and DropIndex on the white list? There may be some LDR specific code we need to add for adding nullable columns without default values.

Reply:
> could we just merge CreateIndex and DropIndex on the white list? There may be some LDR specific code we need to add for adding nullable columns without default values.

not sure i follow. how does handling nullable columns relate to merging create/drop index in the allowlist?

Reply:
sorry, what i mean to say is: lets only include AddIndex and DropIndex on the white list for now.

Reply:
sounds good!